packages/lit-dev-server/src/redirects.ts

```diff
@@ -5,25 +5,47 @@
  */
 
 // prettier-ignore
```

Comment:
Did you intentionally unformat this? I felt it was much more reasonable to do column alignment, hence the `// prettier-ignore` comment.

Reply:
Not intentionally; apparently the prettier-ignore didn't work 🤷

Reply:
Will try and fix.